escrow/free_call_api.go

```diff
@@ -0,0 +1,73 @@
+package escrow
+
+import (
+	"fmt"
+	"math/big"
+)
+
+// To Support Free calls
+type FreeCallPayment struct {
+	//Has the Id of the user making the call
+	UserId string
+	//Service Id .
+	ServiceId string
+	//Organization Id
+	OrganizationId string
+	//Current block number
+	CurrentBlockNumber *big.Int
```

Comment:
This was there to check if the signature has not expired, historic and all current calls from.dapp sign this way, this will need to remain for backward compatibility with dapp.. new signatures check will be in. A seprate PR